README.md

```diff
@@ -80,3 +80,50 @@ pnpm dev
 ```
 
 This will launch the development server and you can access your application at `http://localhost:3000`.
+
+### Folder Structure
```

Comment:
Genel bi not olarak, dizinlerin olası ihtiyaçların tamamına cevap verebilmesi için kurgulandığı, ihtiyaç olmayanların kaldırılmasının faydalı olacağını da ekleyebiliriz. services, store, msw, stories, storybook, mocks gibi.. mock server ihtiyacı yok ise msw ve mocks dizinleri ve ilgili dependency'lerin kaldırılmasını hatırlatabiliriz. template da geldiği için en baştan kullanılmayacağı kesin olsa bile kaldırılmaması gerektiği düşünülebiliyor.